mc/1.20.5/src/main/java/dev/u9g/minecraftdatagenerator/generators/MaterialsDataGenerator.java

```diff
@@ -84,8 +83,9 @@ public static List<MaterialInfo> getGlobalMaterialInfo() {
 
         Registry<Item> itemRegistry = DGU.getWorld().getRegistryManager().get(RegistryKeys.ITEM);
         itemRegistry.forEach(item -> {
-            if (item instanceof MiningToolItem toolItem) {
-                item.getComponents().get(DataComponentTypes.TOOL).rules()
+            ToolComponent toolComponent = item.getComponents().get(DataComponentTypes.TOOL);
```

Comment:
this seems like an unimportant change, unless you can think of a reason to do so it seems unimportant?